make/test.mk

```diff
@@ -304,3 +307,11 @@ else
 		sed -e 's|REPLACE_IMAGE|${IMAGE_NAME}|g' ${E2E_REPO_PATH}/deploy/operator.yaml | oc apply -f -
     endif
 endif
+
+.PHONY: display-eval
+display-eval:
+	@echo 'export HOST_NS=$(shell oc get projects -l app=host-operator --output=name --no-headers=true -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | sort | tail -n 1)'
+	@echo 'export MEMBER_NS=$(shell oc get projects -l app=member-operator --output=name --no-headers=true -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | sort | tail -n 1)'
```

Comment:
```suggestion
	@echo 'export MEMBER_NS=$(shell oc get projects -l app=member-operator --output=name | cut -f2 -d"/" | sort | tail -n 1)'
```

Reply:
same here, keeping the `jsonpath` for now, but indeed, the `--no-headers=true` is not needed, so I'll remove it.

Reply:
done in 689670f